# Urban compost microbiome

Introduction
Urban composting plays a critical role in sustainable agriculture, organic waste recycling, and soil health. Compost systems located near farms and communities also intersect with food safety and public health. This project investigates how microbial communities - including bacteria, fungi, and bacteriophages - change over time during a 6-week heat compost cycle.
Using paired physicochemical analyses and long-read Nanopore metagenomics, we profiled independent urban compost piles across mesophilic and thermophilic stages. Our integrative approach enables longitudinal recovery of compost chemistry and microbiome, improved antimicrobial resistance (AMR) gene detection, novel bacteriophage discovery, and detailed tracking of microbiome succession.

### Preprint link:
https://www.biorxiv.org/content/10.64898/2026.02.10.705224v1

## Repository Contents
This repository is structured to provide all the necessary resources for replicating the analyses presented in our paper. Below is an overview of the contents:

### 1. Code
Project_Codes/: This directory contains all the scripts used for visualizing compost chemistry and microbiology. 

### 2. Data
Project_Data/: Contains chemistry and microbiome data.

## Getting Started
### Prerequisites
To run the analyses, you will need the following software installed on your system:

For nanopore data acquisition:
MinKNOW v23.11.2
MinKNOW core v5.8.0

For rapid nanopore data analysis:
EPI2ME Labs v23.11.2
Dorado v7.2.11 

For taxonomic and AMR analysis:
Kraken2 v2.1.3
Abricate v1.0.1

For phage analysis:
PhaBOX v2.1
PhageScope

For data analysis and visualization:
Python v3.6.4
R v4.5.1 

### Contributors
 - [Angel Montes](https://github.com/angel-montes20)
 - [Hyunjin Shim](https://github.com/hjshim)

### Contributing
We welcome contributions from the community! Please feel free to open an issue or submit a pull request if you have any suggestions or improvements.

### License
This repository is licensed under the GNU GPL License. See the LICENSE file for details.

### Contact
For questions or inquiries about the study, please contact [jinenstar@gmail.com].

We hope this repository serves as a valuable resource for researchers interested in nanopore sequencing and metagenomic analysis. Thank you for your interest in our work!
